package.json

```diff
@@ -1,40 +1,61 @@
 {
- "name": "amp-wp",
- "title": "AMP for WordPress",
- "homepage": "https://github.com/ampproject/amp-wp",
- "repository": {
-  "type": "git",
-  "url": "https://github.com/ampproject/amp-wp.git"
- },
- "license": "GPL-2.0+",
- "private": true,
- "devDependencies": {
-  "@wordpress/dom-ready": "2.0.2",
-  "@wordpress/i18n": "3.1.1",
-  "babel-core": "6.26.3",
-  "babel-loader": "7.1.5",
-  "babel-plugin-transform-object-rest-spread": "6.26.0",
-  "babel-plugin-transform-react-jsx": "6.24.1",
-  "babel-plugin-transform-runtime": "6.23.0",
-  "babel-preset-env": "1.7.0",
-  "cross-env": "5.2.0",
-  "eslint": "4.19.1",
-  "eslint-config-wordpress": "2.0.0",
-  "eslint-plugin-react": "7.11.1",
-  "grunt": "1.0.2",
-  "grunt-contrib-clean": "1.1.0",
-  "grunt-contrib-copy": "1.0.0",
-  "grunt-shell": "2.1.0",
-  "grunt-wp-deploy": "1.2.1",
-  "timeago.js": "3.0.2",
-  "uuid": "3.3.2",
-  "webpack": "4.29.5",
-  "webpack-cli": "3.2.3"
- },
- "main": "blocks/index.js",
- "scripts": {
-  "build": "grunt build && grunt create-build-zip",
-  "deploy": "grunt deploy",
-  "dev": "cross-env BABEL_ENV=default webpack --watch"
- }
+  "name": "amp-wp",
+  "title": "AMP for WordPress",
+  "homepage": "https://github.com/ampproject/amp-wp",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ampproject/amp-wp.git"
+  },
+  "license": "GPL-2.0+",
+  "private": true,
+  "devDependencies": {
+    "@babel/cli": "7.2.3",
+    "@babel/core": "7.3.3",
+    "@babel/plugin-proposal-object-rest-spread": "7.3.2",
+    "@babel/plugin-transform-react-jsx": "7.3.0",
+    "@babel/preset-env": "7.3.1",
+    "@wordpress/babel-plugin-import-jsx-pragma": "1.1.3",
+    "@wordpress/babel-preset-default": "3.0.2",
+    "babel-eslint": "10.0.1",
+    "babel-loader": "8.0.5",
+    "cross-env": "5.2.0",
+    "eslint": "5.14.0",
+    "eslint-config-wordpress": "2.0.0",
+    "eslint-plugin-react": "7.12.4",
+    "grunt": "1.0.2",
+    "grunt-contrib-clean": "1.1.0",
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-shell": "2.1.0",
+    "grunt-wp-deploy": "1.2.1",
+    "npm-run-all": "4.1.5",
+    "timeago.js": "3.0.2",
+    "uuid": "3.3.2",
+    "webpack": "4.29.5",
+    "webpack-cli": "3.2.3"
+  },
+  "main": "blocks/index.js",
+  "scripts": {
+    "build": "npm-run-all build:*",
+    "build:prepare": "grunt build",
+    "build:zip": "grunt create-build-zip",
+    "deploy": "grunt deploy",
+    "dev": "cross-env BABEL_ENV=default webpack --watch",
+    "lint": "npm-run-all --parallel lint:*",
+    "lint:js": "eslint .",
+    "lint:php": "vendor/bin/phpcs",
+    "test": "npm-run-all --parallel test:*",
+    "test:php": "phpunit"
+  },
+  "dependencies": {
+    "@wordpress/blocks": "6.0.7",
```

Comment:
@swissspidy Question: Is this dependency not a throw-away? If Webpack is configured to treat `@wordpress/blocks` among the `externals`, then is there any purpose in making it a dependency other than for IDEs? Why shouldn't these then be `devDependencies`?

Reply:
Semantically they‘re still dependencies, but I get your point. I would need to dig a bit into this to see if there are any big pros or cons to it. Otherwise I don‘t have a string opinion on this.